integration-tests/config.ts

```diff
@@ -91,32 +91,33 @@ const kathmandunetEphemeral = {
   protocol: Protocols.PtKathman,
   signerConfig: {
     type: SignerType.EPHEMERAL_KEY as SignerType.EPHEMERAL_KEY,
-    keyUrl: 'https://api.tez.ie/keys/kathmandunet',
+    keyUrl: 'http://key-gen-1.i.tez.ie:3000/kathmandunet',
     requestHeaders: { Authorization: 'Bearer taquito-example' },
   }
 };
 
-const jakartanetEphemeral = {
-  rpc: process.env['TEZOS_RPC_JAKARTANET'] || 'https://jakartanet-archive.ecadinfra.com',
+const limanetEphemeral = {
+  rpc: process.env['TEZOS_RPC_LIMANET'] || 'https://rpc.limanet.teztnets.xyz',
```

Comment:
Consider using Ecad node instead

Reply:
yes! was just looking for that url :D

Reply:
The one you used below should be fine: https://limanet-archive.ecadinfra.com